control_data_collecting_tool/config/course_param/reversal_loop_circle_param.yaml

```diff
@@ -0,0 +1,25 @@
+/**:
+  ros__parameters:
+    # Course Specific Parameters
+    trajectory_radius: 35.0
+    enclosing_radius: 40.0
```

Comment:
memo: the data collection area only indicates the center, currently need to set in params